### PR TITLE
14275 created folder for completeness report files

### PR DESCRIPTION
**Pull request by saricicekibrahim**

### CODE REVIEW

File: report/completeness.html [hunk 1]
```diff
@@ -89,7 +89,7 @@ <h1 id="countryHeader">${selectCountry}</h2>
 
     async function getStatus(_country, _countryValue) {
         fetchError = "No data for selected country!"
-        await fetch("completeness_" + _countryValue + ".json").then((response) => {
+        await fetch("../completeness_report/completeness_" + _countryValue + ".json").then((response) => {
```
Comment (lgudyma-kontur) [inline report/completeness.html:92]: where do You want this dir to be? `../` in dir is very confusing
better `report/completeness_report/completeness_...`

File: scripts/create_completeness_report.py [hunk 2]
```diff
@@ -73,7 +75,7 @@ def match_datsets(record, files, geoextent_all):
                 and (record[FLD_THEME] == '*' or theme == record[FLD_THEME]) \
                 and (record[FLD_SOURCE] == '*' or record[FLD_SOURCE] == source)\
                 and geometry_type == record[FLD_TYPE]:
-            #print (dataset_name & dateTime)
+            # print (dataset_name & dateTime)
```
Comment (kalenikaliaksandr) [inline scripts/create_completeness_report.py:78]: let's avoid having commented code.

Comment (saricicekibrahim) [reply scripts/create_completeness_report.py:78]: I tried to add explanation. I'll correct there.

File: report/completeness.html (hunk 1 above)
Comment (saricicekibrahim) [reply report/completeness.html:92]: agreed on using completeness_report folder.
js can reach by ../completeness_report

**Pull request merged**